2022q4/Makefile

```diff
@@ -31,7 +31,7 @@ Documentation=	doceng.adoc
 
 # Changes affecting the Ports Collection, whether sweeping changes that
 # touch most of the tree, or individual ports themselves
-Ports= FreshPorts.adoc xfce.adoc pantheon.adoc budgie.adoc gcc.adoc biology.adoc
+Ports= FreshPorts.adoc alderlake-i915kms-drmkmod-update.adoc xfce.adoc pantheon.adoc budgie.adoc gcc.adoc biology.adoc
```

Comment:
alderlake-i915kms-drmkmod-update.adoc pulled after email discussion with both developers of duplicate work.